### DISCUSSION TITLE
Progressive disclosure / hiding sidebar features until applicable (Pull Payments, Payouts, etc..)

### DISCUSSION THREAD

**Discussion by dstrukt** [btcpayserver/btcpayserver#3651] [Ideas / Feature Requests]
Wanted to raise this as a discussion topic (will likely also be discussed on a design call if not addressed here) regarding when to expose certain features to a new store instances.

When you create a new store instance, and have not followed the 3 step setup guide, but navigate to Pull Payments, Payouts, etc.. you're met with this alert...

<img width="1785" alt="Screen Shot 2022-04-18 at 8 34 22 AM" src="https://user-images.githubusercontent.com/6250771/163854204-09158fef-ae3d-46b1-af5a-5782b3840777.png">

Does it make sense to start showing features that can be utilized once certain conditions are met, as opposed to allowing the user to navigate to them to be met with an error/alert?

I wanted to raise the issue to get general thoughts before diving into potential solutions & gauge the general sentiment towards this functionality.

**Comment by bolatovumar** [comment]
On one hand it makes sense maybe that we don't show the feature that user can't use yet but on the other hand it's nice to be able to see what feature are available.

**Comment by dstrukt** [comment]
Yeah, we discussed both of those trade-offs, ultimately ended up redirecting the user back to the Dashboard, that would then push you back through the wallet/node setup.

Might re-explore more at a future time, but was a solid and simple solution that satisfied both criteria.
